xds/internal/resolver/serviceconfig.go

```diff
@@ -93,12 +95,13 @@ func serviceConfigJSON(activeClusters map[string]*clusterInfo) (string, error) {
 }
 
 type route struct {
-	action wrr.WRR
-	m      *compositeMatcher // converted from route matchers
+	m                 *compositeMatcher // converted from route matchers
+	clusters          wrr.WRR
+	maxStreamDuration time.Duration
 }
 
 func (r route) String() string {
-	return r.m.String() + "->" + fmt.Sprint(r.action)
+	return r.m.String() + "->" + fmt.Sprint(r.clusters)
```

Comment:
Done.  I thought about this earlier but forgot to do it.

I think the formatting looks good:
```
pathPrefix:/foo -> { clusters: [{A 1}], maxStreamDuration: 5s }

pathPrefix: -> { clusters: [{cluster_1 75} {cluster_2 25}], maxStreamDuration: 0s }
```

I needed to go into the WRRs to add `String` methods to clean this up (I broke the formatting in my previous PR since it was a map and became a WRR instance).